scripts/rpc/fiat-filter

```diff
@@ -0,0 +1,7 @@
+# param extrinsic (0x9101040...) is a call to did::new()
```

Comment:
Is this file needed?

Reply:
Not strictly.
Only helpful for manual testing.